src/csvReaderWrapper.js

```diff
@@ -28,8 +28,21 @@ const isRowValid = (row) => {
   return cells.every(isCellValid)
 }
 
+const trimWhiteSpaces = (row) => {
```

Comment:
supernit: whitespace is usually used as a collective noun so this can just be `trimWhitespace`.